### PR TITLE
feat(button): 按钮组件样式修改

### PR DESCRIPTION
**Pull request by MNZhu**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/ui-vue/blob/main/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/button/vars.less [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+  --ti-mobile-button-bg-color-danger: #f04b3d;
+  --ti-mobile-button-bg-color-disabled: #e8e8e8;
+  --ti-mobile-button-bg-color-info: #333;
+  --ti-mobile-button-height-medium: 36px;
+  --ti-mobile-button-font-size-medium: 14px;
+  --ti-mobile-button-height-small: 28px;
+  --ti-mobile-button-font-size-small: 12px;
+  --ti-mobile-button-height-mini: 22px;
+  --ti-mobile-button-font-size-mini: 10px;
+  --ti-mobile-button-radius: 2px;
```
Comment (chenxi-20) [inline src/button/vars.less]: border-radius属性变量建议添加border-radius关键字

Comment (MNZhu) [reply src/button/vars.less]: 已删除此项未使用变量定义

File: src/button/index.js [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
+  "tiny-mobile-button-bg-color-info": "#333",
+  "tiny-mobile-button-height-medium": "36px",
+  "tiny-mobile-button-font-size-medium": "14px",
+  "tiny-mobile-button-height-small": "28px",
+  "tiny-mobile-button-font-size-small": "12px",
+  "tiny-mobile-button-height-mini": "22px",
+  "tiny-mobile-button-font-size-mini": "10px",
+  "tiny-mobile-button-text-color": "#4a79fe",
+  "tiny-mobile-button-text-color-hover": "#6e94fe",
+  "tiny-mobile-button-text-color-active": "#4a79fe",
```
Comment (zzcr) [inline src/button/index.js:23]: 你的vscode自动化格式配置，和我们默认的不一样，需要统一一下，我们都是使用单引号

File: src/button/vars.less [hunk 3]
```diff
@@ -1,25 +1,24 @@
...
+  --ti-mobile-button-text-color-white: #fff;
+  --ti-mobile-button-text-color-disabled: var(--ti-mobile-common-color-text-weaken-disabled, #dbdbdb);
+  --ti-mobile-button-text-color-default: var(--ti-mobile-common-color-text-primary, #191919);
+  --ti-mobile-button-border-color: #dbdbdb;
+  --ti-mobile-button-bg-color-default: #fff;
+  --ti-mobile-button-bg-color-primary: var(--ti-mobile-common-color-link-highlight, #4a79fe);
+  --ti-mobile-button-bg-color-success: #52c41a;
+  --ti-mobile-button-bg-color-warning: #faad14;
+  --ti-mobile-button-bg-color-danger: #f04b3d;
+  --ti-mobile-button-bg-color-disabled: #e8e8e8;
```
Comment (zzcr) [inline src/button/vars.less]: 建议这些颜色值都抽成一些基础变量--ti-mobile-common-xxx

Comment (MNZhu) [reply src/button/vars.less]: 已整改

File: src/button/index.js (hunk 2 above)
Comment (MNZhu) [reply src/button/index.js:23]: 已整改

**Review by zzcr — DISMISSED** [review]

**Review by zzcr — APPROVED** [review]

**Pull request merged**